backend/v2/epochs/router.py

```diff
@@ -0,0 +1,85 @@
+from fastapi import APIRouter
+
+from v2.epochs.dependencies import (
+    GetEpochsContracts,
+    GetCurrentEpoch,
+    GetIndexedEpoch,
+    GetRewardsRate,
+)
+from v2.epochs.schemas import (
+    CurrentEpochResponse,
+    IndexedEpochResponse,
+    EpochRewardsRateResponse,
+)
+
+api = APIRouter(prefix="/epochs", tags=["Epochs"])
+
+
+@api.get("/current")
+async def get_current_epoch_v1(
+    epochs_contract: GetEpochsContracts,
+) -> CurrentEpochResponse:
+    """
+    Returns the current epoch number from the blockchain.
+
+    This endpoint queries the smart contract to get the current active epoch number.
+    The epoch number is used throughout the system to track allocation periods
+    and rewards distribution cycles.
```

Comment:
It might be usefull to describe the difference beteween current and indexed. At least the theoretical. 
So something like if they can be the same and when, and when they can differ and why.

Reply:
added the difference in get_indexed_epoch_v1